Check english in docs (#128)
python-microservices/pyms: docs/configuration.md

}
```

This file can contain the following keywords:

## pyms - services block

```pyms```: all subsets inside this keyword are the settings of this library. Each keyword will be a service of our
[Microservice class](ms_class.md). For example, if we declare our microservice class as:

```python
from pyms.flask.app import Microservice
ms = Microservice(path=__file__)
```
and have a `config.yaml` file such as:

```yaml
pyms:
  services:
    requests: true
```

our `ms` object will have an attribute `requests` that is a instance of our service [requests](services.md). 

## pyms - config block
This section contains all keywords used for general [Flask Configuration Handling](http://flask.pocoo.org/docs/1.0/config/), along 

> with our constants for the different the enviroments (local configuration, staging configuration...). Keep in mind that 

Reviewer: ```suggestion
with our constants for the different enviroments (local configuration, staging configuration...). Keep in mind that 
```

a Flask app configuration needs the keywords to be declared as uppercase. If you defined a variable like `app_name`, 

> you will be able to retrieve it this with `current_app.config["APP_NAME"]`

Reviewer: ```suggestion
you will be able to retrieve it with `current_app.config["APP_NAME"]`
```

## Looking for Configuration file with Kubernetes Configmaps

> By default, a Microservice class searches for a config.yml in the same path. You can set a different route or set a json file.

Reviewer: ```suggestion
By default, the Microservice class searches for a config.yml in the same path. You can set a different route or set a json file.
```

> To change this path, you must define a environment variable `CONFIGMAP_FILE`.

Reviewer: ```suggestion
To change this path, you must define an environment variable called `CONFIGMAP_FILE`.
```
